src/Illuminate/Testing/TestResponse.php

```diff
@@ -1149,6 +1150,17 @@ public function assertSessionMissing($key)
         return $this;
     }
 
+    /**
+     * Assert that the response is a given resource.
+     *
+     * @param JsonResource $resource
```

Comment:
Please use the fully qualified class name.

Reply:
fixed